multiclusterhub/charts/search-prod/templates/cm-search-redis-ca-crt.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+  name: search-ca-crt
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822